mlir/include/mlir/Dialect/ArmSVE/IR/ArmSVE.td

```diff
@@ -236,6 +243,66 @@ def UmmlaOp : ArmSVE_Op<"ummla",
     "$acc `,` $src1 `,` $src2 attr-dict `:` type($src1) `to` type($dst)";
 }
 
+
+class SvboolTypeContraint<string lhsArg, string rhsArg> : TypesMatchWith<
+      "expected corresponding svbool type widened to [16]xi1",
```

Comment:
It should be possible to test for this diagnostic, right?

Reply:
This is already implicitly tested. This is used to infer the the svbool type (from the result or argument), so there's no textual representation of an op that fails this constraint.